src/fontman/font_manager.cpp

```diff
@@ -63,19 +79,101 @@ static bool             g_fontManagerIsInit = false;
 typedef std::unordered_map<std::string, int> FontsHash;
```

Comment:
This is an unexpectedly expensive data structure (might easily be 1 KB each even with a small number of fonts loaded).

Reply:
Now the question: what is more memory-effective? The hash-table (unordered_map) or a red-black tree (map)?

Reply:
... trying to find something like: https://stackoverflow.com/questions/9375450/stdunordered-map-very-high-memory-usage

Reply:
I made std::map being used on low-mem ends, should be better. Right?